### PR TITLE
pnp native APIs changelog update

### PR DESCRIPTION
**Pull request by wduraes**
Adding IoT Plug and play native APIs information to changelog file

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,6 +1,29 @@
 # Release History
 
 ## 1.1.0-beta.1 (Unreleased)
+### New Features
+- Public preview version of a new set of APIs to simplify the experience using IoT Plug and Play:
+  - az_iot_pnp_client_init
+  - az_iot_pnp_client_get_user_name
+  - az_iot_pnp_client_get_client_id
+  -	az_iot_pnp_client_sas_get_signature
```
Comment (danewalton) [inline CHANGELOG.md:9]: should these be indented (the dash `-`)

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,6 +1,29 @@
 # Release History
 
 ## 1.1.0-beta.1 (Unreleased)
+### New Features
+- Public preview version of a new set of APIs to simplify the experience using IoT Plug and Play:
+  - az_iot_pnp_client_init
```
Comment (danewalton) [inline CHANGELOG.md:6]: nit: not sure whether I would rather see this here and elsewhere
```suggestion
  - az_iot_pnp_client_init()
```
i guess I'll let you make the call

File: CHANGELOG.md [hunk 3]
```diff
@@ -1,6 +1,29 @@
...
+  - az_iot_pnp_client_get_user_name
+  - az_iot_pnp_client_get_client_id
+  -	az_iot_pnp_client_sas_get_signature
+  -	az_iot_pnp_client_sas_get_password
+  -	az_iot_pnp_client_telemetry_get_publish_topic
+  -	az_iot_pnp_client_commands_parse_received_topic
+  -	az_iot_pnp_client_commands_response_get_publish_topic
+  -	az_iot_pnp_client_property_parse_received_topic
+  -	az_iot_pnp_client_property_document_get_publish_topic
+  -	az_iot_pnp_client_property_patch_get_publish_topic
```
Comment (danewalton) [inline CHANGELOG.md:16]: i feel like tabs are being used here. the rest of this doc uses spaces so just double check this uses spaces?

File: CHANGELOG.md [hunk 4]
```diff
@@ -1,6 +1,29 @@
 # Release History
 
 ## 1.1.0-beta.1 (Unreleased)
+### New Features
+- Public preview version of a new set of APIs to simplify the experience using IoT Plug and Play:
```
Comment (danewalton) [inline CHANGELOG.md:5]: should we mention the header file which these are contained in?
`az_iot_pnp_client.h`

Comment (danewalton) [reply CHANGELOG.md:5]: and maybe the cmake target?
`az::iot::pnp`

**Review by danewalton — APPROVED** [review]

File: CHANGELOG.md (hunk 1 above)
Comment (wduraes) [reply CHANGELOG.md]: fixed!

File: CHANGELOG.md (hunk 3 above)
Comment (wduraes) [reply CHANGELOG.md]: done. had to open this on notepad to find the tabs...

File: CHANGELOG.md (hunk 4 above)
Comment (wduraes) [reply CHANGELOG.md]: added the reference to header file. not sure what do you mean by the other. can you show me what would you add on that?

Comment (danewalton) [reply CHANGELOG.md]: Yup here is the section in the CMake which defines that:
https://github.com/Azure/azure-sdk-for-c/blob/3d9f1a3d530ef5fa12cf481ce7e6a4b798e286d5/sdk/src/azure/iot/CMakeLists.txt#L52-L73

You can see at the very end where it defines the target name `az::iot::pnp` which is what someone would link to if they had a CMake project ingesting these.

File: CHANGELOG.md (hunk 3 above)
Comment (danewalton) [reply CHANGELOG.md]: you can also use VSCode and turn on boundary whitespaces. makes it pretty easy to find.
image

**Review by RickWinter — APPROVED** [review]

**Pull request merged**